README.md

```diff
@@ -73,10 +73,6 @@ Installation instructions for each package are provided in that packages subfold
 
 ### 3. [Grafana](grafana/README.md)
 
-## Usage
```

Comment:
Also need to remove references to these from `## Contents` section up above

Reply:
Fixed